### DISCUSSION TITLE
HOW TO BUY

### DISCUSSION THREAD

**Discussion by KEZEO** [JoomCoder-com/JoomCCK#149] [Q&A] [answered]
hello how to buy the new joomcck ? will it compatible with the JoomVideos Pro Edition and joomcrm please ? need to buy 3

**Comment by Giorgi625** [comment]
Hello here you can buy JoomCCK
https://www.joomcoder.com/joomla-extensions/9-components/24-joomcck

**Comment by andiamine** [comment, marked as answer]
Hi my friend, JoomCCK is free, and there is an extended version to buy here:
https://www.joomcoder.com/joomla-extensions/9-components/24-joomcck

currently we are focusing on making JoomCCK very stable , and also to fork Emerald and make it work with it, after that we are going to add many integrations with Joomla 3rd party extensions.

**Comment by LordHelmchen666** [comment]
Hi @andiamine 
I can not register and buy because Captcha is not visible. Tried with Chrome and Firefox.

**Comment by Giorgi625** [comment]
I just tested with my second email and was able to register. There is no captcha just few fields to fill. I clicked Subscribe now on this page https://www.joomcoder.com/joomla-extensions/9-components/24-joomcck# then Register and filled out fields it redirected me to payment page. 
Did you get captcha problem after this or before?
I tried with Opera GX

**Reply by LordHelmchen666** [reply]
now it was working. suddenly Recaptcha pops up. Strange thing.
